src/components/App.js

```diff
@@ -1,28 +1,164 @@
+  const [allOrders, setAllOrders] = useState([]);
 
   useEffect(() => {
-    getSomething()
-      .then(response => {
-        setMessage(response.message);
+    const currentUser = localStorage.getItem('user')
+    const currentToken = localStorage.getItem('token')
+    if(currentUser && currentToken){
+      const loggedInUser = JSON.parse(currentUser)
+      const loggedInUserToken = JSON.parse(currentToken)
+      setUser(loggedInUser)
+      setToken(loggedInUserToken)
+    }
+  }, []);
+
+  const fetchProducts = () => {
+    getAllProducts()
+      .then( responseAllProducts => {
+        setAllProducts(responseAllProducts)
+      console.log('responseAllProducts: ', responseAllProducts);
+      })
+  }
+
+  const fetchOrders =() => {
+    getAllOrders()
+      .then( responseAllOrders => {
+        setAllOrders(responseAllOrders)
+      console.log('responseAllOrders: ', responseAllOrders);
       })
-      .catch(error => {
-        setMessage(error.message);
-      });
-  });
-
-  return (
-    <div className="App">
-      <h1>Hello, World!</h1>
-      <h2>{ message }</h2>
+  }
+
+  useEffect(()=>{
+    fetchProducts()
+    fetchOrders()
+  },[])
+
+  return <>
+    <div id="App">
+      <Route>
+        <Nav 
+          token = {token}
+          setToken = {setToken}
+          user = {user}
+          setUser = {setUser}
+        />
+      </Route>
+
+      <div className="welcomeDiv">Welcome to Dope Soap!<br />
+        Enjoy a clean view of all our products!
+      </div>
+     
+      {!token
+      ?
+      <Fragment>
+        <Route path="/Login">
+          <Login
+            setUser = {setUser}
+            setToken = {setToken}
+          />
+        </Route>
+
+        <Route path="/Register">
+          <Register
+            setUser = {setUser}
+            setToken = {setToken}
+          />
+        </Route>
+      </Fragment>
+      :
+     <div>
+      <Fragment>
+        <Route path="/SingleUser">
+          <Register
+            setUser = {setUser}
+            setToken = {setToken}
+            singleUser = {singleUser}
+            setSingleUser = {setSingleUser}
+          />
+        </Route>
+      </Fragment>
+     </div>
+      }
+
+      {
+      orderId && userId
+      ?
+      <Fragment>
+        <Route path="/orders">
+          <SingleOrder
+          singleOrder = {singleOrder}
+          setSingleOrder = {setSingleOrder}
+          />
+        </Route>
+      </Fragment>
+      :
+          <span></span>
+      }
+
+      <Route path="/AllOrders">
+        <AllOrders
+        allOrders = {allOrders}
+        setAllOrders = {setAllOrders}
+        />
+      </Route>
+
+      <Route path="/orders/cart">
+        <Cart/>
+      </Route>
+
+      <Route exact path={["/allProducts", "/Home"]}>
+        <AllProducts
+          allProducts = {allProducts}
+          setAllProducts = {setAllProducts}
+         
+        />
+      </Route>
+
+      <Route path={`/allProducts/:productId`}>
```

Comment:
What happens when a product ID doesn't match a product? I.e. `/allProducts/8`. If the api call fails with an error, you could display that error to the frontend!